packages/app-democracy/src/Referendum.tsx

```diff
@@ -66,14 +66,14 @@ class Referendum extends React.PureComponent<Props, State> {
     const newState: State = democracyReferendumVoters.reduce((state, { balance, vote }) => {
       if (vote.valueOf() === true) {
         state.voteCountYay++;
-        state.votedYay = state.votedYay.add(balance.toBn());
+        state.votedYay = state.votedYay.add(balance);
```

Comment:
Nice, Balance now extends BN (in `@polkadot/types`), less hoops to jump through.